plugins/src/webrtcsink/imp.rs

```diff
@@ -213,8 +215,8 @@ struct PipelineWrapper(gst::Pipeline);
 // Structure to generate GstNavigation event from a WebRTCDataChannel
 #[derive(Debug)]
 struct NavigationEventHandler {
-    channel: WebRTCDataChannel,
-    message_sig: glib::SignalHandlerId,
+    _channel: WebRTCDataChannel,
+    _message_sig: glib::SignalHandlerId,
```

Comment:
Why the underscores?

Reply:
Because that new structure introduced warnings I didn't notice at the time of merging. This structure is basically only used to keep a reference on its two members, do you have a suggestion for doing that more elegantly btw? :)

Reply:
`#[allow(dead_code)]` with a comment explaining why

Reply:
heh, works too but not the elegance I was hoping for :)

Reply:
I think the important point is to document why things are strange :)